checks/evaluation/security_policy.go

```diff
@@ -38,12 +39,15 @@ func SecurityPolicy(name string, dl checker.DetailLogger, r *checker.SecurityPol
 			Offset: f.Offset,
 		}
 		if msg.Type == checker.FileTypeURL {
-			msg.Text = "security policy detected in org repo"
+			msg.Text = "security policy detected in org global repo"
```

Comment:
"org repo" seems just fine instead of "org global repo"

Reply:
Yes, I agree with this and have already set it back to the original one ;P